src/test/scala/akka/stream/contrib/SampleSpec.scala

```diff
@@ -0,0 +1,47 @@
+package akka.stream.contrib
+
+import akka.actor.ActorSystem
+import akka.stream._
+import akka.stream.scaladsl.Source
+import org.scalatest.{Matchers, WordSpec}
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class SampleSpec extends WordSpec with Matchers
+{
+  private implicit val system       = ActorSystem("SampleTest")
+  private implicit val materializer = ActorMaterializer()
+
+  "Sample Stage" should {
+    "returns every Nth element in stream" in {
+      val list = 1 to 1000
+      val source = Source.fromIterator[Int](() => list.toIterator)
+
+      for (n <- 1 to 100) {
+        val future = source.via(Sample(n)).runFold(List.empty[Int])((list, e) => e :: list)
+        val expected = list.filter(_ % n == 0).reverse.toList
+
+        Await.result(future, 200 millis) should ===(expected)
```

Comment:
Don't use that short timeouts if you don't have a good reason. CI on Travis can be very slow. 3 seconds is a good default.